releasenotes/notes/k8s-auth.yaml

```diff
@@ -1,7 +1,8 @@
 apiVersion: release-notes/v2
 kind: feature
 area: security
-releaseNotes: |
-       Support migration and concurrent use of regular K8S tokens as well as new K8S tokens with audience. This feature is enabled by 
+releaseNotes:
+- |
+       Support migration and concurrent use of regular K8S tokens as well as new K8S tokens with audience. This feature is enabled by
```

Comment:
Are the spaces required here? Other notes don't have large spaces.

Reply:
Also is it ok to append "**Added**" for security notes?

Reply:
Thanks. Removed the spaces. Went through and looked at the content to make sure that all release notes currently display with the addition of support for action as well.

Reply:
The note above is a release note from the security section and should follow the release notes. Vulnerability notes would appear in the securityNotes section and have their own format: https://github.com/istio/istio/blob/master/releasenotes/README.md#security-notes